boefjes/boefjes/plugins/kat_chrome_mitmproxy/main.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from typing import List, Tuple, Union
+
+import docker
+
+from boefjes.job_models import BoefjeMeta
+from boefjes.plugins.helpers import get_file_from_container
+
+logger = logging.getLogger(__name__)
+
+OCI_IMAGE = "ghcr.io/minvws/nl-kat-chrome-crawler-mitmproxy:latest"
```

Comment:
Could we pin this to a version once we have this PR merged, and a version build and uploaded? While I understand the flexibility in building the image from the latest upstream chome version, I'd want to avoid a situation where the day to day usage of a boefje changes based on a new version of the used container if we can.